elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java

```diff
@@ -390,7 +400,7 @@ private void bindRelation(AccessibleObject fieldOrMethod, String fieldName, Clas
 
         relationshipsDeque.push(fieldName);
         fieldsToValues.put(fieldName, fieldOrMethod);
-        fieldsToTypes.put(fieldName, fieldType);
+        fieldsToTypes.put(fieldName, fieldType == null ? Void.class : fieldType);
```

Comment:
Same here.

Reply:
I get a NPE when fieldType is null.

Reply:
Is it probably the generics that causes the null fieldType.

https://github.com/yahoo/elide/blob/f720136a3bffa03ee90d1fbe5f52a44d57c3d6dd/elide-example-models/src/main/java/com/yahoo/elide/models/generics/Peon.java#L23

Reply:
That is exactly the problem. RawType of type `T` is null.

https://github.com/yahoo/elide/blob/f720136a3bffa03ee90d1fbe5f52a44d57c3d6dd/elide-core/src/main/java/com/yahoo/elide/core/EntityBinding.java#L484